platform/darwin/src/MGLTileID.m

```diff
@@ -0,0 +1,42 @@
+//
+//  MGLTileID.m
+//  ios
+//
+//  Created by Anna Johnson on 2/7/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
```

Comment:
what's up with this 2017 copyright?

Reply:
Anna originally wrote this code 2 years ago, but mapbox-gl has changed so much since then that the originally branches were totally un-mergeable, so the code in this PR is not original, it's copy pasted, though half of this is actually new code.